### PR TITLE
852 - Improve H5P 360 integration to allow opening to a specific scene

### LINKED ISSUE #852: Improve H5P 360 integration to allow opening to a specific scene

### PR DESCRIPTION
**Pull request by wynnset**
## Changes
* Add authoring feature to allow user to select scene if selected H5P is 360 Virtual Tour
* If a scene has been selected by the author, switch to that scene when viewing the H5P
* Some random PHP linting also included that was not before linted
## Screenshot
* This is a space to include a screenshot or GIF if this is a frontend change. Delete this section if this is not a frontend change.
image
## Issue Linkage
Closes #852 
## PR Dependency
Depends on: N/A
## Automated Testing
* No H5P tests added. Any suggestions?

### CODE REVIEW

**Review by shirleywbi — COMMENTED** [review]
@wynnset Given a 360 node, should I be able to navigate from scene to scene within the node? Or is it only one scene per node?

Also, a few styling issues:
image
* Do we want that full screen to be under the favourite/close button?
* It looks like these scenes are elevated (black bar on bottom).

image
* Should this scroll bar be hidden?

**Comment by wynnset** [comment]
> @wynnset Given a 360 node, should I be able to navigate from scene to scene within the node? Or is it only one scene per node?

If the 360 has scenes, you should be able to navigate to it using the built-in h5p 360 functionality (arrow).
image

> Also, a few styling issues:
> * Do we want that full screen to be under the favourite/close button?

That's ok.
> * It looks like these scenes are elevated (black bar on bottom).

I fixed this now, thanks.
> * Should this scroll bar be hidden?

It should be. Is it not hidden for you?

File: templates/vue/src/components/Lightbox/media/H5PMedia/H5PIframe.vue [hunk 1]
```diff
@@ -18,7 +18,7 @@
       width="100%"
       frameborder="0"
       :src="node.typeData && node.typeData.mediaURL"
-      :scrolling="type === 'H5P.InteractiveVideo' && 'no'"
+      :scrolling="library === 'H5P.InteractiveVideo'"
```
Comment (shirleywbi) [inline templates/vue/src/components/Lightbox/media/H5PMedia/H5PIframe.vue:21]: @wynnset What's the rationale for scrolling behaviour in an H5P.InteractiveVideo?

**Review by shirleywbi — APPROVED** [review]
@wynnset Thanks for the clarification. Not sure if we want to keep the scrolling in there for H5P **videos** but otherwise, the new functionality looks good!

**Comment by wynnset** [comment]
@shirleywbi scrolling should be fixed now. Can you check it now? If it's all good, please mark as 'ready for review'

**Review by narendrasss — APPROVED** [review]

**Pull request merged**